service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storageContainer/VerifyAzureStorageContainerCanBeCreatedStep.java

```diff
@@ -27,22 +35,29 @@
  * CreateAzureStorageContainerStep} to ensure idempotency of the create operation.
  */
 public class VerifyAzureStorageContainerCanBeCreatedStep implements Step {
```

Comment:
This seems mis-named. It is locating the storage account for the create, right?

Reply:
Yes, this is validation step which validates pre-requisites for storage container creation. 

Prerequisites: 

1. "Current scenario" - Just storage account.
2. "New scenario" - Landing zone and existing shared storage account in it. 

From my point of class's name reflects behavior well.